dd-sdk-android-core/transitiveDependencies

```diff
@@ -1,15 +1,15 @@
 Dependencies List
 
-androidx.annotation:annotation-experimental:1.1.0               :   16 Kb
-androidx.annotation:annotation:1.3.0                            :   30 Kb
+androidx.annotation:annotation-experimental:1.3.0               :   35 Kb
+androidx.annotation:annotation-jvm:1.8.2                        :   54 Kb
 androidx.arch.core:core-common:2.1.0                            :   10 Kb
 androidx.arch.core:core-runtime:2.1.0                           :    5 Kb
-androidx.collection:collection:1.1.0                            :   41 Kb
-androidx.lifecycle:lifecycle-common:2.1.0                       :   21 Kb
-androidx.lifecycle:lifecycle-livedata-core:2.1.0                :    8 Kb
-androidx.lifecycle:lifecycle-livedata:2.1.0                     :   10 Kb
-androidx.startup:startup-runtime:1.0.0                          :   18 Kb
-androidx.work:work-runtime:2.8.1                                : 1626 Kb
+androidx.collection:collection-jvm:1.4.3                        :  770 Kb
+androidx.lifecycle:lifecycle-common:2.5.1                       :   23 Kb
+androidx.lifecycle:lifecycle-livedata-core:2.5.1                :    9 Kb
+androidx.lifecycle:lifecycle-livedata:2.5.1                     :   11 Kb
+androidx.startup:startup-runtime:1.1.1                          :   18 Kb
+androidx.work:work-runtime:2.9.1                                : 1795 Kb
```

Comment:
So maybe we should stay with `2.8.1` to avoid bringing coroutines? or maybe we can exclude them if they are not used in our code paths?

This is just an idea, I'm curious to see more opinions.

Reply:
I dont want to exclude any of dependencies, because we could create unstable build this way. Rolled back all androidX libs that could add coroutines as a transitive dependency. But we will face this problem in a future anyway